### PR TITLE
s/datagov-deploy/data.gov/ in URLs to reflect new repository name

### PR DESCRIPTION
**Pull request by mogul**

### CODE REVIEW

**Comment by lukefretwell** [comment]
Thank you!

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -331,4 +331,4 @@ If approved, you may merge immediately or leave it to the author. A single
 approval is all that is needed to merge.
 
 
-[datagov-deploy]: https://github.com/GSA/datagov-deploy
+[datagov-deploy]: https://github.com/gsa/data.gov
```
Comment (nickumia-reisys) [inline CONTRIBUTING.md]:
```suggestion
[data.gov]: https://github.com/gsa/data.gov
```

File: Jenkinsfile [hunk 2]
```diff
@@ -93,7 +93,7 @@ pipeline {
       archiveArtifacts artifacts: 'datagov-deploy.tar.gz', onlyIfSuccessful: true
```
Comment (nickumia-reisys) [inline Jenkinsfile:93]: I don't know if this should be changed or how it's generated.

File: docs/roadmap.md [hunk 3]
```diff
@@ -28,7 +28,7 @@ Cloud migration          | <img src="https://img.shields.io/badge/status-started
 
 ### Continuous Integration
 
-[datagov-deploy](https://github.com/GSA/datagov-deploy) is a huge repo and
+[datagov-deploy](https://github.com/gsa/data.gov) is a huge repo and
```
Comment (nickumia-reisys) [inline docs/roadmap.md]:
```suggestion
[data.gov](https://github.com/gsa/data.gov) is a huge repo and
```

**Review by nickumia-reisys — CHANGES_REQUESTED** [review]
The last quick thing I checked was the heading of the README, I can't suggest a change to that because it wasn't modified.

⛳

File: ansible/requirements.yml [hunk 4]
```diff
@@ -40,31 +40,31 @@
   src: https://github.com/GSA/ansible-os-ubuntu-16
   version: 1.0.4
 - name: gsa.datagov-deploy-dashboard
-  src: https://github.com/GSA/datagov-deploy-dashboard
+  src: https://github.com/gsa/data.gov-dashboard
```
Comment (mogul) [inline ansible/requirements.yml]:
```suggestion
  src: https://github.com/gsa/datagov-deploy-dashboard
```

**Review by mogul — COMMENTED** [review]
Missed this one...

File: Jenkinsfile (hunk 2 above)
Comment (mogul) [reply Jenkinsfile:93]: I limited my changes only to places that reference the URL. I don't believe we need to change this one.

**Review by nickumia-reisys — DISMISSED** [review]
🌕

**Review by nickumia-reisys — APPROVED** [review]
🌞

**Pull request merged**